evaluator/src/main/kotlin/com/xebia/funcional/xef/evaluator/TestBuilder.kt

```diff
@@ -0,0 +1,63 @@
+package com.xebia.funcional.xef.evaluator
+
+import com.xebia.funcional.xef.evaluator.models.OutputDescription
+import kotlin.jvm.JvmSynthetic
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class TestBuilder(val description: String, val metric: String) {
```

Comment:
Just a comment. Should this class be changed to `SuiteBuilder` or something similar? It sounds strange to me that the name `TestBuilder` is not a proper test itself.